code/config.ss

```diff
@@ -179,6 +179,13 @@
       (difficulty . 4)
       (topics string cipher))
 
+     ((slug . affine-cipher)
+      (uuid . "c8969b0a-232a-4c88-ab2a-0047e8b4b6d0")
+      (core . #f)
+      (unlocked-by . anagram)
```

Comment:
It might be nice if atbash cipher were core, then it could unlock both `rotational` and `affine`.

Reply:
Yeah, seems possibly a good idea. `atbash` requires a bit of finicky logic though, which would likely make it unpleasant to review student solutions

Reply:
I like the exercise, so once again it is one that I am willing to mentor often.  There are a number of tracks that include it in the core as an exercise to unlock other ciphers.